framework/src/convergence/IterationCountConvergence.C

```diff
@@ -50,15 +50,15 @@ IterationCountConvergence::checkConvergence(unsigned int iter)
       {
         if (_converge_at_max_iterations)
         {
-          oss << "Converged due to iterations (" << iter << ") >= max iterations ("
-              << _max_iterations << ") and 'converge_at_max_iterations' = 'true'.";
+          oss << COLOR_GREEN << "Converged due to iterations (" << iter << ") >= max iterations ("
```

Comment:
There was some argument about setting colors directly in ActionWarehouse.C line 268
```
  /**
   * Note: This routine uses the XTerm colors directly which is not advised for general purpose
   * output coloring.
   * Most users should prefer using Problem::colorText() which respects the "color_output" option
   * for terminals
   * that do not support coloring.  Since this routine is intended for debugging only and runs
   * before several
   * objects exist i
  ```
@permcody you wrote that do we still care?

Reply:
It might have been an old routine